README.md

```diff
@@ -13,6 +13,7 @@ Cloud Foundry deployment.
 * `space`: *Required.* The space to push the application to.
 * `skip_cert_check`: *Optional.* Check the validity of the CF SSL cert.
   Defaults to `false`.
+* `verbose`: *Optional.* Invoke `cf` cli using `CF_TRACE=true` to print all API calls made to Cloud Foundry.
```

Comment:
Why source configuration and not out's parameter?
I think user want to select this setting for each deploy.

Reply:
Hi, that was also my first instinct (you can see that I actually tried it in the commit sequence). However, our use case with the verbose/CF_TRACE setting is to debug stability related issues in Cloud Foundry beta deployments, so we actually want to turn this on by source rather than by app and that means less code in the pipeline.

Reply:
I had a misunderstanding.
It was not to debug for each deploy but to debug the stability of Cloud Foundry beta.

I understand the reason for this.
Thank you for answering:yum: